razorpay-payments.php

```diff
@@ -361,16 +355,28 @@ function getOrderCreationData($orderId)
         /**
          * Generates the order form
          **/
-        function generateOrderForm($redirectUrl, $json)
+        function generateOrderForm($redirectUrl, $data)
         {
-            $html = <<<EOT
-<script src="https://checkout.razorpay.com/v1/checkout.js"></script>
-<script>
-    var data = $json;
-</script>
+            wp_enqueue_script('razorpay_wc_script', plugin_dir_url(__FILE__)  . 'script.js',
+                array('razorpay_checkout'));
+
+            wp_localize_script('razorpay_wc_script',
+                'razorpay_wc_checkout_vars',
+                $data
+            );
+
+            wp_register_script('razorpay_checkout',
```

Comment:
Shouldn't we register this script before `razorpay_wc_script` in order for the steps to be in the right logical order, even though they don't necessarily have to be in that order for it to work?